holidays/countries/nepal.py

```diff
@@ -164,39 +168,40 @@ def _add_non_continuous_holidays(self, *, is_workday: bool = False):
             }
             if dt := republic_day_dates.get(self._year):
                 # Established in 2009.
+
                 # Republic Day.
-                self._add_holiday("Republic Day", dt)
+                self._add_holiday(tr("गणतन्त्र दिवस"), dt)
 
             # Hindu Holidays.
 
             # Ram Navami.
-            self._add_ram_navami("Ram Navami")
+            self._add_ram_navami(tr("राम नवमी"))
 
             # Janai Poornima.
-            self._add_raksha_bandhan("Janai Poornima")
+            self._add_raksha_bandhan(tr("जनै पूर्णिमा"))
 
             # Shree Krishna Janmashtami.
-            self._add_janmashtami("Shree Krishna Janmashtami")
+            self._add_janmashtami(tr("श्रीकृष्ण जन्माष्टमी"))
 
             # Ghatasthapana.
-            self._add_sharad_navratri("Ghatasthapana")
+            self._add_sharad_navratri(tr("घटस्थापना"))
 
             # Duwadashi (Dashain).
-            self._add_papankusha_duwadashi("Duwadashi (Dashain)")
+            self._add_papankusha_duwadashi(tr("दुवादशी (दशैं)"))
 
     def _populate_public_holidays(self):
         if self._year >= 2023:
             # Prithvi Jayanti.
-            self._add_holiday_jan_11("Prithvi Jayanti")
+            self._add_holiday_jan_11(tr("पृथ्वी जयन्ती"))
 
         # International Women's Day.
-        self._add_womens_day("International Women's Day")
+        self._add_womens_day(tr("अन्तर्राष्ट्रिय महिला दिवस"))
 
-        # Nepal New Year.
-        self._add_vaisakhi("Nepali New Year")
+        # Nepali New Year.
+        self._add_vaisakhi(tr("नेपाली नयाँ वर्ष"))
 
         # International Labour Day.
```

Comment:
All l10n comments are currently standardised as `en_US`
```suggestion
        # International Labor Day.
```